autoload/commands/__load__

```diff
@@ -66,6 +66,7 @@ do
                     "$fg[green]$repo$reset_color"
                 ;;
         esac
+        exit $?
```

Comment:
I wasn't certain - I wasn't sure if there were any issues with the jobs dying in Bash on Windows and so I added that to have a greater degree of confidence that it would. I think it can be removed though - I'll do that as soon as I can.